ddtrace/contrib/celery/__init__.py

```diff
@@ -0,0 +1,64 @@
+"""
```

Comment:
The documentation is updated here: https://github.com/DataDog/dd-trace-py/pull/531